packages/run-protocol/src/vaultFactory/vaultManager.js

```diff
@@ -254,79 +268,83 @@ const helperBehavior = {
       return;
     }
 
-    const liquidationMargin = state.factoryPowers
-      .getGovernedParams()
-      .getLiquidationMargin();
-
-    // ask to be alerted when the price level falls enough that the vault
-    // with the highest debt to collateral ratio will no longer be valued at the
-    // liquidationMargin above its debt.
-    const triggerPoint = ceilMultiplyBy(
-      highestDebtRatio.numerator, // debt
-      liquidationMargin,
-    );
+    // INTERLOCK: the first time through, start the process to process
+    // liquidations over time.
+    if (!state.liquidationInProgress) {
+      state.liquidationInProgress = true;
+      // eslint-disable-next-line consistent-return
+      return facets.helper
+        .processLiquidations()
+        .catch(e => console.log('Liquidator failed', e))
+        .finally(() => {
+          state.liquidationInProgress = false;
+        });
+    }
 
-    // INTERLOCK NOTE:
-    // The presence of an `outstandingQuote` is used to ensure there's only
-    // one thread of control waiting for a quote from the price authority.
-    // All later callers will just update the threshold for the `outstandingQuote`.
-    // The first invocation will skip the next block and continue to below where
-    // the `outstandingQuote` is assigned. So, if there's an outstanding quote,
-    // reset the level. If there's no current quote (because this is the first
-    // vault, or because a quote just resolved), then make a new request to the
-    // priceAuthority, and when it resolves, liquidate the first vault.
+    // There is already an activity processing liquidations. It may be
+    // waiting for the oracle price to cross a threshold.
+    // Update the current in-progress quote if there is one. Otherwise,
+    // the new threshold will be picked up by the next quote request.
     if (state.outstandingQuote) {
+      const govParams = state.factoryPowers.getGovernedParams();
+      const liquidationMargin = govParams.getLiquidationMargin();
       // Safe to call extraneously (lightweight and idempotent)
       E(state.outstandingQuote).updateLevel(
         highestDebtRatio.denominator, // collateral
-        triggerPoint,
+        liquidationThreshold(highestDebtRatio, liquidationMargin),
       );
-      trace(
-        'updating level for outstandingQuote',
-        triggerPoint,
-        highestDebtRatio.denominator,
-      );
-      return;
+      trace('update quote', highestDebtRatio);
     }
+  },
```

Comment:
if not doing the above, could be clearer that doing nothing in the else is intentional
```suggestion
  // do nothing; the new threshold will be picked up by the next quote request
  },
```